doc/source/suse-ecp/ose-briefing.rst

```diff
@@ -0,0 +1,56 @@
+.. _ose-briefing:
```

Comment:
I think this should be in the overview (ose-overview) instead. Overview already explains those steps, it just not writing the names of the steps, but what they are doing. 

you could write on the overview page that each of the "blocks" on the graph is an independent step in the deployment.

You could then map the "steps" by linking their name and their equivalent argument in run.sh.

Reply:
We considered writing those commands in overview (ose-overview), but in the present documentation the `run.sh` command is introduced later in 'prepare the target hosts' (ose-targets). We also added a brief explanation of that command there.

So introducing the `run.sh` command in the overview would also mean we should explain the preparation needed for it which is already covered in 'prepare localhost' (ose-localhost). For those reasons we added that final step which simplified the whole process.

Reply:
Maybe renaming the "Process summary" would then help. Else I expect the briefing to happen first :)

But that's a nit I guess, so I think we can merge as is.

Reply:
changed section name from "briefing" to "process summary".